### ISSUE TITLE
Keyboard Shortcuts Issue

### ISSUE THREAD

**Issue by Inexpress** [goldendict/goldendict#1549]
Apologies for this very general and nonspecific issue. I've noticed that when using GoldenDict, sometimes it'll simply stop responding to my Control+C+C input, and I won't be able to use that to pull up the dictionary window. Upon restarting the software it works fine, but having to do so is quite frustrating. Any suggestions for how I could go about troubleshooting this issue?

Many thanks for this excellent software.

**Comment by vedgy** [comment]
What operating system and GoldenDict version do you use?

**Comment by Inexpress** [comment]
Windows 10 current release, GoldenDict 1.5.0-RC2-372-gc3ff15f

**Comment by vedgy** [comment]
0cea62699d3aeee77d9c363028303ba6e591ae2c worked around a similar Ctrl+C+C issue on GNU/Linux. The same or similar workaround might help on Windows too. But someone (not me, I don't use windows) has to make a Windows build with the workaround applied for testing.

**Comment by Inexpress** [comment]
I don't have the expertise to comment, but it appears that issue is a race condition one where handling is broken all the time for certain apps. In my case, the shortcut is broken only sometimes, but for all apps until the software is restarted.

**Comment by vedgy** [comment]
Yes, but the workaround is a simple 10 ms delay. Even if the root cause is different, a delay might work it around.

**Comment by wildekat** [comment]
> and I won't be able to use that to pull up the dictionary window

Could you elaborate a bit on that please?
There is a commonly reported problem on Windows where Ctrl-C-C does trigger a lookup, but fails to bring the window to the foreground. Would that be your case by any chance?

**Comment by Inexpress** [comment]
I just checked when the issue recurred, and that doesn't seem to be the case—the word in question isn't visible in GoldenDict's word history list.

**Comment by wildekat** [comment]
OK, it must be a different problem then.
But anyway, you might want to try the workaround and see if it makes a difference: exit GoldenDict, move GdHotkeys.dll out of the way (e.g. rename it to GdHotkeys.dll.bak), start GoldenDict.

**Comment by Inexpress** [comment]
The error doesn't occur all the time, only after a while of using the software. I'll follow the steps you suggest, and see if the error recurs.

**Comment by kanjieater** [comment]
I had the same issue on windows and reverted to an older version of GdHotkeys.dll to fix it. https://github.com/goldendict/goldendict/issues/1156

**Comment by neowit** [comment]
> OK, it must be a different problem then. But anyway, you might want to try the workaround and see if it makes a difference: exit GoldenDict, move GdHotkeys.dll out of the way (e.g. rename it to GdHotkeys.dll.bak), start GoldenDict.

Thank you wildkat, this workaround works perfectly. It's been 2 weeks since I implemented it (Win 10 Pro 21H2, GoldenDict 1.5.0-RC2-372-gc3ff15f) and have had no issues with keyboard shortcuts since.
